### PR TITLE
Viewer: ignore some node modules. Slims down bundle.

### PR DESCRIPTION
**Pull request by ebidel**
R: all

This ignores some unneeded modules from the bundle.

screen shot 2016-12-14 at 12 26 23 pm

### CODE REVIEW

**Comment by brendankenny** [comment]
Is log never used? Don't we map it to console.log in the browser anyways?

**Comment by ebidel** [comment]
Don't think it's needed. When testing the page I didn't see anything that needed it. Errors from this app are routed to the butter bar.

**Comment by brendankenny** [comment]
looks like none of the files currently use it, so that works, but we could easily add a `log.log()` message to report-generator at some point and it'll cause errors that won't necessarily be easy to track down. It's [pretty small](https://github.com/GoogleChrome/lighthouse/blob/847a28d3a61e08b03bb417de6e65a5e80ac6f9a1/lighthouse-core/lib/log.js) and even smaller after minifying or whatever

**Comment by brendankenny** [comment]
also, do you want to add a [build step like for the extension in travis.yml](https://github.com/GoogleChrome/lighthouse/blob/master/.travis.yml#L31-L33) while you're adding the fatal eslint thing?

File: lighthouse-viewer/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "eslint-config-xo": "^0.10.1",
```
Comment (brendankenny) [inline lighthouse-viewer/package.json]: don't need `eslint-config-xo` anymore

**Comment by ebidel** [comment]
Added the travis building. I'm going to keep the logger module out for now. We know we don't need it and anything that gets added upstream could potentially cause errors here. Need to do some tests!

**Comment by brendankenny** [comment]
error on travis:

```sh
$ cd lighthouse-viewer
The command "cd lighthouse-viewer" exited with 0.
$ gulp
/home/travis/build.sh: line 57: ./node_modules/.bin/gulp: No such file or directory
The command "gulp" exited with 127.
$ cd ..
```

looks like you probably need to npm install/yarn yarn

File: .travis.yml [hunk 2]
```diff
@@ -31,5 +33,8 @@ script:
   - cd lighthouse-extension
   - gulp build -- $__node_harmony
   - cd ..
+  - cd lighthouse-viewer
+  - gulp
```
Comment (brendankenny) [inline .travis.yml]: for future reference, you might need to add `-- $__node_harmony` (like the extension does) if you end up using fancier features in your build process

Comment (ebidel) [inline .travis.yml]: word

**Pull request merged**